### ISSUE TITLE
Wifi disconnecting randomly using iwctl

### ISSUE THREAD

**Issue by ohxxm** [AsahiLinux/docs#14]
OS: Arch Linux ARM Minimal

Wifi Manager: iwctl

No error is outputted

**Comment by Mase3206** [comment]
Yeah, I noticed this occasionally as well. I've been able to _temporarily_ fix it by just reconnecting over and over until it stops disconnecting.

**Comment by h0m3** [comment]
I'm on NetworkManager and I don't experience this issue so far. It may be specific to iwd.

**Comment by Spaxly** [comment]
I have switched to NetworkManager and have had no issue

**Comment by Mase3206** [comment]
It must have been fixed in an update of NetworkManager, ‘cause I’m not experiencing it anymore. Weird. 
>

**Comment by monomycelium** [comment]
Is `iwd` set as the back-end for NetworkManager? See https://wiki.archlinux.org/title/NetworkManager#Using_iwd_as_the_Wi-Fi_backend.

**Comment by Mase3206** [comment]
I have no idea. I switched to Ubuntu a month or two ago, because I am not very familiar or comfortable with Arch when compared to Ubuntu, and I did not like KDE. Since I’ve switched to Ubuntu, I haven’t seen any issues regarding WiFi. Plus, now that I’m not using Arch anymore, I can’t offer a lot of help. My bad… I didn’t even know this issue was still open! On Jan 22, 2023, at 8:49 PM, mashrafi rahman ***@***.***> wrote:﻿
Is iwd set as the back-end for NetworkManager? See https://wiki.archlinux.org/title/NetworkManager#Using_iwd_as_the_Wi-Fi_backend.

**Comment by chadmed** [comment]
If you are still experiencing this, please raise with the Asahi ALARM maintainers

**Comment by mkurz** [comment]
> If you are still experiencing this, please raise with the Asahi ALARM maintainers

Which is here: https://github.com/asahi-alarm/
